Makefile

```diff
@@ -21,9 +21,10 @@ create_mix_project:
 common_addon_prompts = Y\nY\nY\nY\nY\nY\nY\nY\n
 
 # Y - in response to Would you like to add the SVG Sprite addon?
-# Y - in response to Would you like to add the Nimble CSS addon?
+# Y - in response to Would you like to add the Dart Sass addon?
```

Comment:
I think in any web project, we will always need SCSS right so Dart SCSS is a must to compile it and should be a default addon. No?

Reply:
It depends @longnd for example the Micky and Thanh's project is using Tailwind CSS which doesn't require SCSS

Reply:
it makes sense to me now 👍